File: example/lib/render_kit_ui/RenderKitRegistry.swift
import SwiftUI

struct RenderKitRegistry {
    static let screens: [String: ([String: Any], @escaping (String, [String: Any]) -> Void) -> AnyView] = [
        "IncomingCallScreen": { state, onEvent in
            AnyView(IncomingCallScreen(state: state, onEvent: onEvent))
        }
    ]
}

File: android/src/main/kotlin/com/mohammed_fawaz_cp/render_kit_flutter/RenderKitRegistry.kt
package com.mohammed_fawaz_cp.render_kit_flutter

import androidx.compose.runtime.Composable

object RenderKitRegistry {
    val screens = HashMap<String, @Composable (state: Map<String, Any>, onEvent: (String, Map<String, Any>) -> Unit) -> Unit>()
}
